Gruntfile.js

```diff
@@ -461,15 +461,86 @@ module.exports = function (grunt) {
             // Settings that are specific to MV3
             merge(manifestJSON, {
                 action: {
+                    default_popup: 'popup/popup.html',
                     default_icon: {
                         20: config.options.icon16,
                         40: config.options.icon48,
                     },
                 },
+                permissions: [
+                    'alarms',
+                    'notifications',
+                    'scripting',
+                    'storage',
+                    'tabs',
+                    'webNavigation',
+                ],
                 background: {
                     service_worker: 'bundle/serviceWorker.bundle.js',
                 },
-                host_permissions: [],
+                host_permissions: ['*://*/*'],
+                web_accessible_resources: [
+                    {
+                        resources: [
+                            'insights.html',
+                            'assessments/*',
+                            'injected/*',
+                            'background/*',
+                            'common/*',
+                            'DetailsView/*',
+                            'bundle/*',
+                            'NOTICE.html',
+                        ],
+                        matches: ['<all_urls>'],
+                    },
+                ],
+                commands: {
```

Comment:
It looks like the `commands` sections are identical between v2 and v3; should that be moved to the base `manifest.json` instead of being repeated?